react-finger-exercises/react/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,32 @@
 import { isArray } from './utils';
 
 export function min() {
-  isArray();
-  return Math.min();
+  if (arguments === 1) {
```

Comment:
Using rest parameters, this function would be simplified to just one if (checking if there are any arguments).